website/blog/2023-03-10-databend-weekly-84.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "This Week in Databend #84"
+date: 2023-03-10
+slug: 2023-03-10-databend-weekly
+tags: [databend, weekly]
+description: "Learn about the latest updates of Databend in a week!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Carlosfengv
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: drmingdrmer
+  - name: dusx1981
+  - name: everpcpc
+  - name: jun0315
+  - name: leiysky
+  - name: mergify[bot]
+  - name: PsiACE
+  - name: quaxquax
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: xinlifoobar
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> :loudspeaker: This week we announced the official release of Databend v1.0. This is an important milestone, to learn more, please check out our blog: https://databend.rs/blog/databend-release-v1.0
```

Comment:
We announced the milestone release of Databend 1.0 this week. For what's new in the release, please read this post: https://databend.rs/blog/databend-release-v1.0.